### DISCUSSION TITLE
Possibility to tune the name of the page for the paper from Zotero

### DISCUSSION THREAD

**Discussion by abhii-roy** [e-zz/logseq-pdf-extract#12] [Ideas]
The plugin at this stage is very nice, I had the following idea.

Quite often, the name of the research article is very long, which makes the logseq page's name very long. The Zotero plugin,Zot file is quite popular for renaming the PDF flies, which gives customization as to how one would rename the PDF. 

![image](https://github.com/e-zz/logseq-pdf-extract/assets/95222112/553ac345-2f1b-4e89-9f21-6733462a1189)

I believe that if there were an option in the pulgin to automatically name the page names when importing from Zotero based on their metadata, it would significantly enhance the organization and neatness of logseq graph.

**Comment by e-zz** [comment]
Thanks for the idea and explanation. 
Here are my first thoughts about it
- One possibility is to use the directly the name generated by Zotero (with renaming rules defined in Zotero or Zotfile)
- Another way is to let the plugin generate the title based on item metadata. So we need to allow users to maintain another title template apart from the one in Zotero. And this could be a good starting point for the template feature in another discussion.

For now I am not sure which way is better. If the name given by Zotero is what you want, tell me and I think it's possible to include it in the next release. 

And I will update again if I have more info!

--- 

I will try to add the title template feature to this pulgin because of the situation mentioned here https://github.com/logseq/logseq/issues/11328

**Reply by abhii-roy** [reply]
Thank you for your prompt reply.

**Comment by e-zz** [comment]
Hi again. An experimental feature of using Zotero title template has been released in `0.1.1`. Please check https://github.com/e-zz/logseq-pdf-extract/releases/tag/0.1.1 and see if it improves the situation. 

It can be enabled here in the plugin's menu
![image](https://github.com/e-zz/logseq-pdf-extract/assets/37920155/8f750e3e-998e-46fd-98f2-06aad851b03a)
